packages/argo-checkout/src/extension-points/api/post-purchase/post-purchase.ts

```diff
@@ -148,13 +148,6 @@ interface MoneyBag {
   presentmentMoney: Money;
 }
 
-interface LineItem {
```

Comment:
This was a duplicate of https://github.com/Shopify/argo-checkout/pull/40/files#diff-59da4811d6b641027ac9f174efdbdcf4R75.